pkg/controller/reconciler.go

```diff
@@ -66,8 +66,16 @@ func doReconcile(ks *k8sclient.K8sClient, kc *kubeletClient) {
 		mit := newMountInfoTable()
 		podList, err := kc.GetNodeRunningPods()
 		if err != nil {
-			klog.Errorf("doReconcile GetNodeRunningPods: %v", err)
-			goto finish
+			klog.Warningf(`failed to GetNodeRunningPods from kubelet("%s"), fallback to apiserver`, err.Error())
+			if config.CSIPod.Spec.NodeName != "" {
+				podList, err = ks.ListPodsOnNode(ctx, config.CSIPod.Spec.NodeName)
+			} else {
+				podList, err = ks.ListPodsOnNodeIP(ctx, config.CSIPod.Status.HostIP)
```

Comment:
Polling Apiserver by list pod will cause a lot of pressure on Apiserver, it is recommended to change it to watch.

Reply:
As we need the target pods of mount pods, we cannot avoid listing `Pod` on the `apiserver`.